config/dwdirectiverules/dw_directive_rules.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: dws.cray.hpe.com/v1alpha1
+kind: DWDirectiveRule
+metadata:
+  name: dws-test-driver-directive-rules
```

Comment:
Kustomize is pre-pending "dws-test-driver-" to this name, so it's coming out as "dws-test-driver-dws-test-driver-directive-rules".  That's from the namePrefix value in config/default/kustomization.yaml.

Change it here to just "directive-rules", then it'll come out as "dws-test-driver-directive-rules".


And  please stop doing the force-pushes.

Reply:
Sorry about the force pushes! I forgot I wasn't on my own fork.